authority/authority.go

```diff
@@ -958,3 +963,40 @@ func (a *Authority) startCRLGenerator() error {
 
 	return nil
 }
+
+//nolint:gocritic // used in defered statements
+func (a *Authority) incrProvisionerCounter(prov *provisioner.Interface, err *error, count func(Meter, string, bool)) {
```

Comment:
This is an interface
```suggestion
func (a *Authority) incrProvisionerCounter(prov provisioner.Interface, err *error, count func(Meter, string, bool)) {
```

Reply:
There's pointers there as this is called only via `defer` statements. I understand it's an interface.